app/validation/validators.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, min=None, max=None, messages=None):
         self.messages = messages
 
     def __call__(self, form, field):
-        data = field.data
+        data = int(field.data)
```

Comment:
I ended up have to use StringFields in the time_input_form. Mainly because of Mandatory means only one field is need out of the 2 (hours  or minutes). If I used IntergerField and I only give it minutes, it throws an error because of the None. I looked at your CustomIntegerField but that cleans the data if its not a valid integer to none, which wouldn't work for me later. So I used stringField, which at some point I need to cast it and this is the only place I could really do it.